src/state/CallViewModel/localMember/LocalTransport.ts

```diff
@@ -154,26 +162,15 @@ async function makeTransportInternal(
       livekit_alias: livekitAlias,
     };
     logger.log("Using LiveKit transport from config: ", transportFromConf);
-    return transportFromConf;
+    transport = transportFromConf;
```

Comment:
is it also returning transport?

Reply:
this needed some update (the priority should have been the other way around) updated it.